.editorconfig

```diff
@@ -0,0 +1,12 @@
+# https://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
```

Comment:
Spaces are preferable

Reply:
Changed indent_style **_tab_** to **_space_**